<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="Tab">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Live_tab">
       <attribute name="title">
        <string>Live mode</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="Add_new_Button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="9">
             <widget class="QSpinBox" name="Top_speed_input">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>120</number>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="9">
             <widget class="QSlider" name="Speed_slider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QToolButton" name="Decrease_speed">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/left-symbolic.svg</normaloff>assets/left-symbolic.svg</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="8">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Top speed</string>
              </property>
             </widget>
            </item>
            <item row="1" column="10">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Km/h</string>
              </property>
             </widget>
            </item>
            <item row="6" column="10">
             <widget class="QToolButton" name="Increase_speed">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/right-symbolic.svg</normaloff>assets/right-symbolic.svg</iconset>
              </property>
             </widget>
            </item>
            <item row="4" column="8" colspan="3">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="9">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" rowspan="6" colspan="6">
             <widget class="QLCDNumber" name="Speed_monitor">
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="smallDecimalPoint">
               <bool>false</bool>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Filled</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="5">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7" rowspan="5">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Km/h</string>
              </property>
             </widget>
            </item>
            <item row="2" column="8" colspan="3">
             <widget class="QPushButton" name="Accept_Top_speed">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/check-symbolic.svg</normaloff>assets/check-symbolic.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>52</height>
               </size>
              </property>
              <property name="text">
               <string>Accelerate</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/speedometer-symbolic.svg</normaloff>assets/speedometer-symbolic.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>52</height>
               </size>
              </property>
              <property name="text">
               <string>Brake</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/brake-symbolic.svg</normaloff>assets/brake-symbolic.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Script_tab">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Script mode</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
